router/router.json

```diff
@@ -2417,6 +2417,16 @@
               "default": false
             }
           },
+          {
+            "name": "partition",
+            "in": "query",
+            "description": "A comma-separated list of values to identify sections of the route that correspond to truck routes, non-truck routes, ferry routes, non-ferry routes and locality names.  The response includes a partitions attribute, which is an array of objects, each of which has an index (into the route coordinate array) and a value for each of the attributes requested in the partition parameter. Any or all of the following values can be used. <br><br>Partition values:<br> isTruckRoute – Distinguish between truck routes and non-truck routes <br> isFerry – Distinguish between ferry routes and non-ferry routes <br> locality – Include the locality name for the route partition",
```

Comment:
```suggestion
            "description": "A comma-separated list of values to identify sections of the route that correspond to truck sections and non-truck sections, ferry sections and non-ferry sections, and locality names.  The response includes a partitions attribute, which is an array of objects, each of which has an index (into the route coordinate array) and a value for each of the attributes requested in the partition parameter. Any or all of the following values can be used. <br><br>Partition values:<br> isTruckRoute – Distinguish between truck sections and non-truck sections <br> isFerry – Distinguish between ferry sections and non-ferry sections <br> locality – Include the locality name for the route partition",
```

Reply:
Updates have been made to use the wording 'truck route sections' as well as 'ferry sections'